Mathlib/Algebra/DirectSum/Internal.lean

```diff
@@ -35,6 +35,16 @@ When `CompleteLattice.Independent (Set.range A)` (a weaker condition than
 mapping `⨁ i, A i →+ ⨆ i, A i` can be obtained as
 `DirectSum.toAddMonoid (fun i ↦ AddSubmonoid.inclusion $ le_iSup A i)`.
 
+This file also provides some extra structure on `A 0`, namely:
+* `SetLike.GradeZero.subsemiring`, which leads to
+  * `SetLike.GradeZero.instSemiring`
+  * `SetLike.GradeZero.instCommSemiring`
+* `SetLike.GradeZero.subring`, which leads to
+  * `SetLike.GradeZero.instRing`
+  * `SetLike.GradeZero.instCommRing`
+* `SetLike.GradeZero.subalgebra`, which leads to
+  * `SetLike.GradeZero.instAlgebra`
```

Comment:
@FMLJohn: I took the liberty of adding the `Comm` versions and the `Algebra` instance.

Reply:
Oh I see! Nice!